.circleci/main.yml

```diff
@@ -144,14 +144,16 @@ jobs:
         path: /tmp/circleci-test-results
   sharness:
     machine:
-        image: ubuntu-2004:202010-01
+        image: ubuntu-2204:2022.10.1
```

Comment:
This is not required. I only updated Ubuntu version as I was trying to investigate runtime differences between CircleCI and GitHub Actions and wanted to make the environment more similar. It proved to have no affect on runtime.